src/browser/session.ts

```diff
@@ -79,39 +79,33 @@ class Session extends EventEmitter {
 
                     case 7:
                         reason = 'lock';
-
-                        // NOTE: when the screen is locked, the machine is considered idle.
-                        // there is no need for the checkIdleStateTimer until the screen is unlocked
-                        this.checkIdleStateTimer.stop();
-
-                        if (!this.idleEventTimer.isRunning()) {
-                            this.idleStartTime = app.getTickCount();
-                            this.fireIdleEvent(true);
-                            this.idleEventTimer.reset();
-                        }
+                        this.dealwithLock();
```

Comment:
What're your thoughts on consolidating the locking and unlocking code into a single function?
```ts
class Session extends EventEmitter {
  private handleLock(locked: boolean): void {
    if (locked) {
      // code for handling locked state
    } else {
      // code for handling unlocked state
    }
  }
}
```
Otherwise, I think renaming the functions to `handleLock()` and `handleUnlock()` is fine enough.